docs/run-keep-ecdsa.adoc

```diff
@@ -323,6 +323,65 @@ reference].
 |No
 |===
 
+==== Example BeneficiaryAddresses
+
+[%header,cols=2*]
+|===
+|Description
+|Address
+
+|mainnet xpub extended public key
+|`BeneficiaryAddress = "xpub6Cg41S21VrxkW1WBTZJn95KNpHozP2Xc6AhG27ZcvZvH8XyNzunEqLdk9dxyXQUoy7ALWQFNn5K1me74aEMtS6pUgNDuCYTTMsJzCAk9sk1"`
+
+|mainnet ypub extended public key
+|`BeneficiaryAddress = "ypub6Xxan668aiJqvh4SVfd7EzqjWvf36gWufTkhWHv3gaxnBh44HpkTi2TTkm1u136qjUxk7F3jGzoyfrGpHvALMgJgbF4WNXpoPu3QYrqogMK"`
+
+|mainnet zpub extended public key
+|`BeneficiaryAddress = "zpub6rePDVHfRP14VpYiejwepBhzu45UbvqvzE3ZMdDnNykG47mZYyGTjsuq6uzQYRakSrHyix1YTXKohag4GDZLcHcLvhSAs2MQNF8VDaZuQT9"`
+
+|testnet3/regtest tpub extended public key
+|`BeneficiaryAddress = "tpubDEXzoXkNdhoFeYrtS2BJKfok6LwH5PKkr5jPSMR6A2erw2yS3VgY5EoYdcKH24VPqeAgBTF6i82Ft9NG1iVjSQVAvFBfd2wkRQXF1W2Q8W1"`
+
+|testnet3/regtest upub extended public key
+|`BeneficiaryAddress = "upub5DnYQWgCDSGe6DAr6NZaD4jsh1Qz55jh7DeScVVyJJnHCp5BwyvM7Xm7S5r5n5ZYMJ1WrrM31i4kcsWwW2vxcJS1kfsuKgK9vME2z1cx6aX"`
+
+|testnet3/regest vpub extended public key
+|`BeneficiaryAddress = "vpub5Zx5difzitDBNPjrr9pTno6C44dJFd89naYzhyk9QWHFTpF7pJqnyAnADhbVrFYX7eCK8V2WBBVprxzJrSk15NsYHiB8CvV8h4JnXkU66as"`
+
+|standard mainnet P2PKH btc address
+|`BeneficiaryAddress = "1MjCqoLqMZ6Ru64TTtP16XnpSdiE8Kpgcx"`
+
+|standard testnet P2PKH btc address
+|`BeneficiaryAddress = "mkHS9ne12qx9pS9VojpwU5xtRd4T7X7ZUt"`
+
+|standard mainnet P2SH btc address
```

Comment:
In some descriptions, we've got `standard` as the first word. Is this relevant? Could we remove it to have all descriptions consistent?